.github/workflows/ci.yml

```
@@ -36,7 +36,7 @@ jobs:
           # to `docker run` and bind-mount an /etc/passwd that maps that to `developer`.
           # We also need write permissions for the group
           chmod -R g+w . && sudo chgrp -R 1000 . && make format && sudo chgrp -R $(id -g) .
-          make openapi
+          make -C lambdas/ openapi
```

```suggestion
          make -C lambdas openapi
```

I would only do this when it makes a difference, like for `rsync` or `ln -s foo bar/`

https://unix.stackexchange.com/a/7795/448602

I may have given contrary advice a while back.